nim-mode.el

```diff
@@ -134,7 +134,11 @@
   (setq-local electric-indent-inhibit t)
   (setq-local electric-indent-chars '(?: ?\s))
   (when electric-indent-mode
-    (define-key nim-mode-map [remap delete-backward-char] 'nim-electric-backspace)))
+    (define-key nim-mode-map [remap delete-backward-char] 'nim-electric-backspace))
+
+
+  ;; Set `compile-command' to the correct initial value
+  (nim-compile--set-compile-command))
```

Comment:
I am sorry, I don't get it. AFAIK I can't just copy past the line (nim-compile--set-compile-commend) as a top level statement in ``nim-compile.el`` that would mean something different.